src/nimony/nimony.nim

```diff
@@ -157,18 +158,25 @@ proc handleCmdLine() =
     if not isChild:
       createDir(config.nifcachePath)
       createDir(binDir())
+      # configure required tools
+      setCurrentDir(compilerDir())
```

Comment:
I don't like this solution at all. Isn't there a better way?

Reply:
i think i can tweak the `requiresTool` to find the src folder properly like it does with binaries, and add something in semos to execute `exec "git submodule update --init"` always in compiler path